skalenetwork/sgxwallet
# Feature/438 ubuntu 22

fixes https://github.com/skalenetwork/sgxwallet/issues/438

added support of ubuntu22 and g++-11
updated `sgx sdk` and `sgx_driver`
updated libBLS version
update README and build instructions

to test:
- run the old-version sgxwallet on ubuntu20 machine
- stop the container, copy the database to the ubuntu22 server
- start the new-version sgxwallet with `-b` flag
- ensure that all keys are accessible and produces valid signatures / public keys
